docker/owlbot/python/cloudbuild.yaml

```diff
@@ -0,0 +1,10 @@
+steps:
+    - name: 'gcr.io/cloud-builders/docker'
+      args: [ 'build',
+        '-t', 'gcr.io/repo-automation-bots/owlbot-python:$SHORT_SHA',
+        '-t', 'gcr.io/repo-automation-bots/owlbot-python:latest',
+        '-f', 'docker/owlbot/nodejs/Dockerfile', '.' ]
+    - name: 'gcr.io/cloud-builders/docker'
+      args: ['push', 'gcr.io/repo-automation-bots/owlbot-python:$SHORT_SHA']
+    - name: 'gcr.io/cloud-builders/docker'
+      args: ['push', 'gcr.io/repo-automation-bots/owlbot-python:latest']
```

Comment:
Consider using the [`images:`](https://cloud.google.com/build/docs/build-config#images) section of cloudbuild.yaml

Reply:
Done.